spec/redmine_adapter_mock.rb

```diff
@@ -1,6 +1,6 @@
 class RedmineAdapterMock
   #<TimeEntry id: 1815, project_id: 24, user_id: 1, issue_id: 916, hours: 0.2325, comments: "84561435: Intern - Sitzung - Nächste Sitzung vorber...", activity_id: 45, spent_on: "2013-07-30", tyear: 2013, tmonth: 7, tweek: 31, created_on: "2013-07-30 14:40:32", updated_on: "2013-07-30 14:40:32">
-  TimeEntryMock = Struct.new(:issue_id, :comments, :activity_id, :hours, :id, :saved) do
+  TimeEntryMock = Struct.new(:issue_id, :spent_on, :comments, :activity_id, :hours, :id, :saved) do
```

Comment:
why do you add `spent_on` in front? if you add it at the end, you do not need to change any of the mocks below, right?